package.json

```diff
@@ -62,37 +64,10 @@
     "webpack-cli": "^3.1.2",
     "webpack-dev-server": "^3.1.10",
     "webpack-plugin-hash-output": "^3.1.0",
-    "webpack-shell-plugin": "0.4.5",
+    "webpack-shell-plugin": "https://github.com/cdeutsch/webpack-shell-plugin.git#bee537d",
```

Comment:
doesnt look right

Reply:
[See this](https://github.com/1337programming/webpack-shell-plugin/issues/61#issuecomment-384090619). It's the only way to get rid of the 
```
(node:83992) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
    at WebpackShellPlugin.apply (/projectDir/node_modules/webpack-shell-plugin/lib/index.js:236:16
```
deprecation warning. The original `webpack-shell-plugin` appears discontinued. I don't know if we should rather live with the deprecation warning.